### ISSUE TITLE
Arrange screen based content displays and drivers in the venue

### ISSUE THREAD

**Issue by Tyler-Ward** [srobo/tasks#900]
This ticket is to track the following items

* where display screens are being placed in the venue/are installed in the venue
  * see #737 for acquisition of screens
* what content is displayed on them e.g. livestream/comp info
* what is driving the screen e.g. pi/pc/
* what cables/adapters are required to connect things up inc power extensions ect

**Comment by Tyler-Ward** [comment]
@WillB97 do you have a list of which screens are built into the venue

**Comment by WillB97** [comment]
- large screen on concourse, HDMI in.
- 3-4 screens in the cafe, hdmi in.
- Screen a far end of bar 2, pc connected

**Comment by Tyler-Ward** [comment]
https://github.com/srobo/tasks/issues/170 is also an option for some displays

**Comment by Tyler-Ward** [comment]
Details of the screens are in the floorplans

**Comment by Tyler-Ward** [comment]
Enumerated list of all screens in the venue based on the floorplans

* Cube
  * Display by the doors
  * 4* arena displays
  * 1 staging display
  * 1 by score entry
* L2
  * Screen by cube doors
  * built in screen at end of L2
* L3
  * Screen on bar in American diner area
  * 3x built in displays in cafe
* L4
  * built in video wall

**Comment by PeterJCLaw** [comment]
>     * Cube
>       * Display by the doors
>         * displays: comp-outside
>         * requires: large screen, and display driver with required cable (almost certainly HDMI)

Do we have an actual screen sorted for this? I seem to recall that unlike in the past this isn't something that SUSU can provide, though I don't know where we got to with finding an alternative.

In terms of drivers, SR has 5 * Pi 3B's (one of which is needed as the compbox Pi) and 12 Pi 1As for a total of 16 available drivers. In the past we've used some of the 3Bs as part of the livestream setup (capture side -- four of them have Pi cams I think).

From the above I count 14 places where we need drivers, plus the extras noted at the bottom. That feels like it uses up our supply of Pis completely, which is fine.

In terms of preparation for displaying the livestreams within the venue I suggest we handle this within srcomp-kiosk too. It isn't really SRComp specific currently -- it just maps a MAC address to a URL.
